src/System.Net.Primitives/src/System/Net/HttpStatusCode.cs

```diff
@@ -56,6 +56,7 @@ public enum HttpStatusCode
         ExpectationFailed = 417,
 
         UpgradeRequired = 426,
+        TooManyRequests = 429,
```

Comment:
If adding one status from RFC 6585, it seems reasonable to add the rest (428, 431 & 511) at the same time.

Reply:
It does, though only 429 is causing me a problem. Shall I add that commit?

Reply:
@JonHanna done, cheers for the suggestion :-)